01week/datatypes.js

```diff
@@ -0,0 +1,102 @@
+//Write a JavaScript program to display the current day and time.
+function findTime(){
+const now = new Date(); //find date
+const date = (now.getMonth() + 1) + "/" + now.getDate(); //display month/day
+const time = now.getHours() + ":" + now.getMinutes(); //display hour:minute
+const dayArray = ["Sunday", "Monday", "Tuesday", "Wednesday", "Thursday", "Friday", "Saturday"]; //array for day of the week
+const day = dayArray[now.getDay()]; //get string day of week
+const dateTime = "It is " + day + ", " + date + " at " + time + "."; //display time and date in string
+return dateTime;
+}
+
+findTime();
+
+//Write a JavaScript program to convert a number to a string.
+//Create a funtion to return
+function numToString(num1,num2){
+  return ('The winner is number') + ' ' +(num1 + num2);
+}
```

Comment:
you need to use a method that will convert a number to a string. Right now you are just using the same a parameters and adding them to a string literal.